packages/text-area/theme/material/vaadin-text-area-styles.js

```diff
@@ -21,6 +21,53 @@ const textArea = css`
     white-space: pre-wrap; /* override "nowrap" from <vaadin-text-field> */
     align-self: stretch; /* override "baseline" from <vaadin-text-field> */
   }
+
+  /* Do not show inherited underline from vaadin-text-field */
+  /* See: https://github.com/vaadin/web-components/issues/1333 */
+  [part="input-field"]::before,
+  [part="input-field"]::after {
+    display: none;
+  }
+
+  /* Set the pseudoelement in vaadin-text-area-container instead */
+  /* NOTE: If DOM ever changed, this will break */
+  .vaadin-text-area-container::before,
+  .vaadin-text-area-container::after {
```

Comment:
We should not use internal parts in themes.

Reply:
I am just voting for pragmatism here